packages/cli/test/scripts/push.test.js

```diff
@@ -323,6 +323,19 @@ contract('push script', function([_, owner]) {
     });
   };
 
+  const shouldSaveOwner = function () {
+    it('should save owner when not specifying "from"', async function () {
+      const txParams = {};
+      await push({ networkFile: this.networkFile, network, txParams });
+      should.exist(this.networkFile.owner);
```

Comment:
based on your implementation, this one should check the saved owner is the coinbase address, however, I think the owner is going to be the first address in the list instead of the coinbase